app/src/main/java/com/example/playlistmaker/ui/library/LibraryActivity.kt

```diff
@@ -1,14 +1,36 @@
 package com.example.playlistmaker.ui.library
 
 import android.os.Bundle
-import androidx.activity.enableEdgeToEdge
 import androidx.appcompat.app.AppCompatActivity
 import com.example.playlistmaker.R
+import com.example.playlistmaker.databinding.ActivityLibraryBinding
+import com.google.android.material.tabs.TabLayoutMediator
 
 class LibraryActivity : AppCompatActivity() {
+
+    private val binding by lazy { ActivityLibraryBinding.inflate(layoutInflater) }
```

Comment:
Так как binding будет создаваться всегда на главном потоке, то можно выключить синхронизацию для lazy:
```suggestion
    private val binding by lazy(mode = LazyThreadSafetyMode.NONE) { ActivityLibraryBinding.inflate(layoutInflater) }
```